### PR TITLE
Add CSS for contact validation

### PR DESCRIPTION
**Pull request by dfizzkruah93**
- Update Index file

- Add Js file for form validation

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -297,6 +297,35 @@ document.addEventListener('click', (e) => {
...
+
+  // if the email is valid, proceed with form submission
+  if (valid) {
+    // empty the response test in contact form
+    responseContainer.innerHTML = '';
+  } else {
+    e.preventDefault();
+    responseContainer.innerHTML = 'Invalid Email';
+  }
+});
```
Comment (mmsesay) [inline js/script.js]: - [ ] The email verification is not working as expected. I inputted an uppercase email then the form got submitted without any errors. Please work on this validation to check if the email is in lower case.

See test screenshot

Filling the form | After Submitting |
:------------------:|:----------------------:|
a1| a2

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @dfizzkruah93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @dfizzkruah93,


## Wow, you did it 🎉
image

Brilliant
## Thank you for correcting the form validation, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dfizzkruah93** [comment]
@Meltrust. Thanks for the approval.

**Pull request merged**